Userland/Libraries/LibIMAP/Objects.h

```diff
@@ -320,6 +323,76 @@ class FetchResponseData {
     BodyStructure m_body_structure;
 };
 
+String serialize_astring(StringView string);
+
+struct SearchKey {
+public:
+    // clang-format off
+    struct All{};
```

Comment:
At least properly format them if you're gonna turn off clang-format

Reply:
clang-format puts all the closing braces on a different line which looks super weird (in my opinion, at least). If you'd like I can turn it back on though.

Reply:
I don't mind having them on one line, but please don't eat the spaces : )
e.g. `struct All { };` or `struct Bcc { String bcc; };`